sources/academy/tutorials/node_js/block_requests_puppeteer.md

```diff
@@ -28,8 +28,8 @@ we could can use this piece of code
 ```js
 await page.setRequestInterception(true);
 page.on('request', (request) => {
-            if (someCondition) request.abort();
-            else request.continue();
+    if (someCondition) request.abort();
+    else request.continue();
```

Comment:
i'd rather see curly braces in control statements, its much more readable, especially if you have the else branch. this could be probably also autofixed via eslint.

Reply:
I agree, just didn't go through these manaully. We would need to add a new rule to our lint.